shellcheck/test.sh

```
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+name="world"
+
+echo Hello $name
+
+for file in $(ls *.txt)
```

Shellcheck (0.10.0)

[error] Iterating over ls output is fragile. Use globs.

(SC2045)